lib/rouge/all.rb

```diff
@@ -0,0 +1,5 @@
+Rouge::LOAD_ALL = true
+
+require 'rouge'
```

Comment:
No, not necessarily. As expressed in the OP:

> A consumer can opt-out of the caching behaviour entirely by requiring `rouge/all` rather than `rouge`.

I use it in the specs Rake task but that's honestly because MiniTest's crazy metaprogramming breaks _my_ crazy metaprogramming. Or at least I think that's what was going on. In any event, since the test suite needs to test every lexer, it felt like a legitimate use for, and a good opprotunity to test, `require 'rouge/all'`.